nucliadb_models/nucliadb_models/search.py

```diff
@@ -31,10 +32,10 @@
 from nucliadb_models.metadata import RelationType, ResourceProcessingStatus
 from nucliadb_models.resource import ExtractedDataTypeName, Resource
 
-if TYPE_CHECKING:
-    SortValue = OrderBy.OrderType.V
-else:
-    SortValue = int
+# if TYPE_CHECKING:
+#     ClientTypeValue = ClientType.V
+# else:
+#     ClientTypeValue = int
```

Comment:
does this need to be uncommented back?

Reply:
i'll remove it,  i could't find a way to make that work with enums, it was just breaking in one point of another, so i typ ignored the two offending lines in the stream.py file

Reply:
Fixed